tools/phenomenal/nmr/batman.xml

```diff
@@ -3,67 +3,32 @@
-         <data name="MultipletsPpmShifts" format="tabular" from_work_dir="results/MultipletsPpmShifts.txt" label="A table (M x n) containing the posterior means of multiplets ppm shift for M multiplets as its rows."/>
-         <data name="RelConCreInt" format="tabular" from_work_dir="results/RelConCreInt.txt" label="Relative concentrations in the Confidence Interval of 95%"/>
-
+         <data name="zip_output" format="zip" from_work_dir="results/zip_output.zip" label="BATMAN output files zipped"/>
          <collection name="spectra" type="list" labels="Stack from ${on_string}">
              <discover_datasets pattern="(?P&lt;name&gt;specFit_.+_)\.pdf$" ext="pdf" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;L_.+)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;NMRdata_mod_.+)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;beta_sam_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;beta_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;delta_draw_mean_.+)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;delta_sam_.+)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;lambda_sam_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;metaFit_sam_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;metaIndFit_sam_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;metaTempHR_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;metaTemp_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;specFitHR_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;specFit_.+_rr_0)\.txt$" ext="tabular" directory="results" />
-             <discover_datasets pattern="(?P&lt;name&gt;theta_sam_.+_rr_0)\.txt$" ext="tabular" directory="results" />
          </collection>
      </outputs>
      <help><![CDATA[
 Four mandatory files: NMRdata.txt, batmanOptions.txt, multi_data_user.csv, metabolitesList.csv
-NMRdata.txt is the NMR spectra data for BATMAN to process peak fitting. 
-
-The NMRdata.txt is a tabulated file with the first column of data as p.p.m and the rest of columns as NMR spectra.
+NMRdata.txt is the NMR spectra data for BATMAN to process peak fitting. The NMRdata.txt is a tabulated file with the first column of data as p.p.m and the rest of columns as NMR spectra.
```

Comment:
Ok, I guess that we are not going to get links to example files... I leave this with you for the future. Will approve this PR as I understand that you have tested this locally with minikube and data tests are passing.